src/examples/grid-example.css

```diff
@@ -1,23 +1,23 @@
 .frozen-column {
-  composes: container from '../grid/grid-cell.css';
+  composes: container from '../../packages/grid/src/grid-cell.css';
```

Comment:
For these examples, I just did a normal import up and into the packages directory. Seemed to be the most straightforward.